static/docs/commands-reference/cache.md

```diff
@@ -19,3 +34,15 @@ Modify cache settings.
   no problems arise, otherwise 1.
 
 - `-v`, `--verbose` - displays detailed tracing information.
+
+## Example
+
+The main use of this command is to apply the `-v` and `-q` options to `dvc cache
+dir` which doesn't have them:
```

Comment:
Given this fact ^ do we even need the `dvc cache` command? Would it be possible to only have `dvc cache dir` since there are no other sub-commands? I'll bring this discussion to Discord.

Reply:
https://discordapp.com/channels/485586884165107732/565699007037571084/578648616995127306

Reply:
Ruslan said there will be other subcommands in the future like `dvc cache type`.